packages/yoastseo/src/languageProcessing/researches/findKeyphraseInSEOTitle.js

```diff
@@ -102,37 +238,18 @@ const findKeyphraseInSEOTitle = function( paper, researcher ) {
 	}
 
 	// Check if the exact match of the keyphrase is found in the SEO title.
+	const prefixedFunctionWordsRegex = researcher.getConfig( "prefixedFunctionWordsRegex" );
 	const keywordMatched = wordMatch( title, keyword, locale, false );
 
-	if ( keywordMatched.count > 0 ) {
+	if ( keywordMatched.count > 0 && ! prefixedFunctionWordsRegex ) {
```

Comment:
Skip this check for Arabic and Hebrew. This check is included in `findExactMatch` function